beamer/agent/util.py

```diff
@@ -120,9 +120,9 @@ def make_web3(
     w3.middleware_onion.inject(geth_poa_middleware, layer=0)
     w3.middleware_onion.add(construct_sign_and_send_raw_middleware(account))
 
-    # Cache data of 1000 least recently used blocks.
-    cache_class = cast(Type[dict[Any, Any]], functools.partial(lru.LRU, 1000))
-    middleware = construct_simple_cache_middleware(cache_class=cache_class)
+    # Cache data of 1000 entry.
```

Comment:
Typo: `entry` -> `entries`

Reply:
Can you explain how `SimpleCache` works?

Reply:
I looked at the code. It is a simple OrderedDict with a FIFO queue. So unfortunately not LRU.

Reply:
corrected the typo here.